automated install/basic-install.sh

```diff
@@ -528,8 +528,8 @@ make_repo() {
     printf "  %b %s..." "${INFO}" "${str}"
     # If the directory exists,
     if [[ -d "${directory}" ]]; then
-        # delete everything in it so git can clone into it
-        rm -rf "${directory}"
+        # Return with a 1 to exit the installer. We don't want to overwrite what could already be here in case it is not ours
```

Comment:
We could `printf` something that says `/var/www/html/admin` exists before returning `1`. Just to have some kind out output to look at when users show us "Contact support" screenshots.

Reply:
`printf "${directory}"` I mean.

Reply:
Done, see latest commit